modules/cnf-deploying-the-numa-aware-scheduler.adoc

```diff
@@ -8,62 +8,50 @@
 
 After you install the NUMA Resources Operator, do the following to deploy the NUMA-aware secondary pod scheduler:
 
-* Configure the pod admittance policy for the required machine profile
+* Configure the performance profile. 
 
-* Create the required machine config pool
-
-* Deploy the NUMA-aware secondary scheduler
+* Deploy the NUMA-aware secondary scheduler.
 
 .Prerequisites
 
 * Install the OpenShift CLI (`oc`).
 
 * Log in as a user with `cluster-admin` privileges.
 
+* Create the required machine config pool.
+
 * Install the NUMA Resources Operator.
 
 .Procedure
-. Create the `KubeletConfig` custom resource that configures the pod admittance policy for the machine profile:
+. Create the `PerformanceProfile` custom resource:
 
-.. Save the following YAML in the `nro-kubeletconfig.yaml` file:
+.. Save the following YAML in the `nro-perfprof.yaml` file:
 +
 [source,yaml]
 ----
-apiVersion: machineconfiguration.openshift.io/v1
-kind: KubeletConfig
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
 metadata:
-  name: cnf-worker-tuning
+  name: perfprof-nrop
```

Comment:
@ffromani - I just placed the `annotations` content as a placeholder now. Can you provide some details about the known issue and what the fix does? 
I think it's OK to just include fix as commented lines in this YAML and the user can un-comment if they are encountering the issue?